playbooks/installation/index.adoc

```diff
@@ -242,6 +242,39 @@ The Smart Start architecture requires the following Servers(VMs) be provisioned:
   ** 4 GB Memory
   ** 10 GB Root Drive
 
+=== Ansible Control Host
+
+The OpenShift advanced installer uses Ansible playbooks specifically designed to install OpenShift.  The importance of having a separate host to install your OpenShift Cluster allows for a central location to manage all your OpenShift clusters.  It also provides a jump point into your many clusters.  During the installation process some system processes are restarted and thus if the advanced installer is run from the first master it can cause installation errors.  After the virtual machine is provisioned and on the network, we will need to ensure we can assign the correct repo's so we can install atomic-openshift-utils.
```

Comment:
... It also provides a jump point into your many clusters _that may not sit on a publicly accessible network_.